operator/v1/types.go

```diff
@@ -115,6 +115,7 @@ type OperatorStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
 	// conditions is a list of conditions and their status
+	// +listType=atomic
```

Comment:
This should behave as a map
```suggestion
	// +listType=map
	// +listMapKey=type
```